sdk/src/test/java/io/dapr/it/BaseIT.java

```diff
@@ -27,6 +27,7 @@
   @BeforeClass
   public static void setEnvironmentVariables(){
     environmentVariables.set("DAPR_HTTP_PORT", String.valueOf(DAPR_FREEPORTS.getHttpPort()));
+    environmentVariables.set("DAPR_GRPC_PORT", String.valueOf(DAPR_FREEPORTS.getGrpcPort()));
```

Comment:
this should be set by dapr cli.

Reply:
no, this is a Mock of the environment variables, is not the actual ones, because, at this moment, IT test does not know about that variables, and actually there is another problem where the client is a singleton, so even if dapr change the port, the client is not aware of that.

Reply:
This code runs outside dapper, so it is needed.